src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -121,7 +121,7 @@ private void validateFullCommonDetails(OverseasEntitySubmissionDto overseasEntit
 
     private void validateTrustDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
         if (isTrustWebEnabled && !CollectionUtils.isEmpty(overseasEntitySubmissionDto.getTrusts())) {
-            trustDetailsValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
+            trustDetailsValidator.validate(overseasEntitySubmissionDto, errors, loggingContext);
             if (!overseasEntitySubmissionDto.isForUpdateOrRemove()) {
                 trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
```

Comment:
(Probably) not an issue, but I don't understand why lines 126 to 128 are only run for a registration. Decision taken by Kainos when API validation for Update was introduced. Would be happy to hear any theories on this...

Reply:
Maybe Andre might remember?

Reply:
@AndreKramer7654 does this ring any bells?

In the meantime, I'm going to add a 'be aware' comment to the code, just to make this a bit clearer.